token/program-2022/src/extension/confidential_transfer/instruction.rs

```diff
@@ -1279,3 +1273,82 @@ pub fn disable_non_confidential_credits(
         multisig_signers,
     )
 }
+
+/// Create a `TransferWithSplitProof` instruction without fee
+#[allow(clippy::too_many_arguments)]
+#[cfg(not(target_os = "solana"))]
+pub fn transfer_with_split_proofs(
+    token_program_id: &Pubkey,
+    source_token_account: &Pubkey,
+    destination_token_account: &Pubkey,
+    mint: &Pubkey,
```

Comment:
nit: I'm just noticing this now, so you don't need to fix it in this PR, but all of the confidential transfer functions order these as "source, destination, mint", whereas the normal transfer functions order it as "source, mint, destination". We should make that consistent at some point

Reply:
Yep, thanks for the catch. I'll reorder the inputs in the confidential transfer functions in a follow-up PR.